editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -1705,6 +1705,11 @@ export const CanvasControlTypeKeepDeepEquality: KeepDeepEqualityCall<CanvasContr
         return KeyboardCatcherControlKeepDeepEquality(oldValue, newValue)
       }
       break
+    case 'FLOW_SLIDER':
+      if (newValue.type === oldValue.type) {
+        return keepDeepEqualityResult(oldValue, true)
```

Comment:
to make sure that we update the code if the flowSlider() factor function ever gets a parameter, in this file we usually add a line calling the factory function in the root with a comment saying if this breaks, update the code. look at lines 1675-1676